apps/lib/apps.c

```diff
@@ -828,6 +828,7 @@ int load_crls(const char *uri, STACK_OF(X509_CRL) **crls,
     return ret;
 }
 
+#define SET_EXPECT(val) expect = expect < 0 ? val : expect == val ? val : 0;
```

Comment:
a comment explaining this would be helpful, as would the redundant parens after the first colon.

Reply:
Done